ScanInfoToGsheet/source/service_account.json

```diff
@@ -1,12 +0,0 @@
-{
-  "type": "service_account",
-  "project_id": "scaninfo-275704",
-  "private_key_id": "95cbe2ad30f1d92df89e6b37be4c58a319d4acfd",
-  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDvJO+488leojEK\nuqLOrhyQllxauMMEoUd7i4+AQU8OSUAyc5MuI1jF0eXS2uUlDIHECMWi4Gmb/U7b\npLriVWf3gypNEHSPOmy8uME731yEtPzWdCQujB2d2GULrQFaMu1hIsaMxTVohsNi\nOAkLV1nUW4miyD0HT4lXyB+7D28ckmtgmP0yntgtdZZHa2cjFbYZirKpda5hMFH4\nzaV/zGXt2loiK4Dv9qIjT0R8ior/LrU9VHa9DQruUqT8PN3uI6YmrVzJo54SKThQ\nkkBZ+LxV9A6A/ReO9YiOlYNufzbaOklNyDajSorxWYd8z1bVLvO4FktYpGr1jyc9\ngTYhPAq/AgMBAAECggEAddr0MWgjs3mnRaxZFBaGiVMwzJ0Fr0B0ZBSRchm9hkLf\nWn3O6QwOcd9Rriqmhy20vhG8pNqH8ZJSzLhQSb7hb/+6VyRm7TdT42YCBSm9BMHV\nYTL3qeDvr5y3n5txcSxsBFUJGNMP0SWpILyQ6H/yC0/ZctnQCcxO2W4NBTlD/wXN\n88Qj/cBBiYGlWAMnS360S0MyjFZh0rcyig26kxQPN/PBH8O6XQRylheTnsHSa/7s\n6Q8DJ3kGP5P325oIhSNwnJAvC5QCtzfP4CWmWISqh12Nk+QsrEUfL4ykd2KYs7lh\nBh9bXOk61JHuu9oiQTkqyVnO5hQj005pwMNCv1T5cQKBgQD6YL7Ei99WRf8s0PIf\n07XccuI9FhGk9AShQ9oBSeF4lZSHZFCefC0QlBwyKCFOZsOHgtjV+eJpKusx8Uqx\nFsJkNAcKH6cE+N4FyArt0b3KCRv29rbQotUvvCIiJyauARyKlFq4EHau/yxIY0el\nXJqZDk05Nm2jDK0JJtp09+PUPQKBgQD0g53bfOFHVN3DQNtpBd3xELLYFgnmIV9T\nzVCQJqWWJAwcrlAco/TRJ6J38NjDwRG/ielviXJVmo0CFQdql8fPAJPftFiGDOSb\nppal0ba+x5qwL+Lg0E6bxvWHIkpYkzjxF12pd8tpciWcdMDrYoSAb3Goqy+l6Saf\ng4iHJyi+qwKBgQCJB8HgmBzXft+1i3W4bYou6MFhZV6qxNJz4ErEQccg09zCbOii\nlJulYyo7WTNBUYTZ+YbrNSPxIwguAT07xpUFemQP6lriocIBsAYrOouAs5OWilhr\nv3Kl+XVE8+B7gyg3wBIWgwwxon5977gajKAf/G/PTDac8icRxy3wrmsGkQKBgQDP\nANpUpl2mYQ+5rJ/UQtZJruRqEgZhYOFudgR9dWp8OJpUcbJ6/MdXhuGB/1HUMBDR\ngnUxK0CzfjtghuMXd9X9xrRqECldm26y447oZSn8S8oG3ROLVm8mbIMS25HreQAd\nA7YUsclwxOVYlnN5E5evkCm+LOcS49cB9n5QRXO2LwJ/etfv+tWmYEerQKk1A+a1\nDICCpOOmB2DzANRiDohf5PNvGSCWDvCrHJhzwn33hbXRMoXF2r2Mz5kxXew5PZ+I\npFtjOIGwLDYDbrYNYG0C2z7XZmyMMaqhz2nED/DnqGUCvkTb4G2BVDNPoFxX5JW9\nonyCF/UBGlhdHDgPpQKFEg==\n-----END PRIVATE KEY-----\n",
```

Comment:
@Fumikac just double-checking: did you also revert this access token in google and created a new one? (the token used here should be revoked.)

Reply:
@ax3l Good point, I just disabled the old one and created a new key. Thanks!